drape_frontend/colored_symbol_shape.cpp

```diff
@@ -109,6 +109,11 @@ void ColoredSymbolShape::Draw(ref_ptr<dp::GraphicsContext> context, ref_ptr<dp::
 
   glsl::vec2 const pt = glsl::ToVec2(ConvertToLocal(m_point, m_params.m_tileCenter,
                                                     kShapeCoordScalar));
+  // TODO: if m_params.m_depthTestEnabled == false then passing a real
+  // m_params.m_depth value to OGL doesn't make sense, but could lead to
+  // elements out of [dp::kMinDepth, dp::kMaxDepth] depth range being
+  // not rendered at all. Depth values are derived from priorities often
+  // hence it leads to unnecessary restriction of possible priorities range.
```

Comment:
Let's put assert here, but these comments are not relevant. These are API classes.